spotipy/client.py

```diff
@@ -339,7 +339,7 @@ def album(self, album_id):
         trid = self._get_id("album", album_id)
         return self._get("albums/" + trid)
 
-    def album_tracks(self, album_id, limit=50, offset=0):
+    def album_tracks(self, album_id, limit=50, offset=0, market=None):
         """ Get Spotify catalog information about an album's tracks
 
             Parameters:
```

Comment:
can update this with the new parameter

Reply:
I've updated the album_tracks docstring--if there's another change needed, please clarify